public/app/percona/integrated-alerting/components/Table/Filter/Filter.styles.ts

```diff
@@ -0,0 +1,48 @@
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getStyles = ({ v1: { colors } }: GrafanaTheme2) => {
+  return {
+    searchSelect: css`
+      margin: 0;
+    `,
+    searchTextInput: css`
+      margin: 0;
+    `,
+    filterWrapper: css`
+      background-color: ${colors.bg2};
+      border: 2px solid ${colors.border2};
+      border-bottom: none;
+      padding: 2px 16px;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+    `,
+    filterLabel: css`
+      font-size: 20px;
+    `,
+    filterActionsWrapper: css`
+      display: flex;
+      justify-content: right;
+      align-items: center;
+      gap: 5px;
+      width: 33%;
+    `,
+    advanceFilter: css`
+      border-top: 2px solid ${colors.border2};
+      padding: 16px;
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      gap: 5px;
+    `,
+    searchFields: css`
+      display: flex;
+      gap: 10px;
+      width: 100%;
+    `,
+    icon: css`
+      margin-top: 5px;
+      margin-bottom: 5px;
+    `,
```

Comment:
I see a lot of hard-coded paddings / margins / borders / ...
Can we try to use values from the theme as much as possible?